refactor: favor CDN for examples (#11)
Esri/arcgis-maps-sdk-javascript-samples-beta: packages/coding-components/templates/webpack/readme.md

yarn run build
```

#### JS

> We imported the components we need for the coding components by following [Stencil's instructions for integrating components without a JavaScript framework](https://stenciljs.com/docs/javascript).

Reviewer: ```suggestion
Import the components using [Stencil's instructions for integrating components without a JavaScript framework](https://stenciljs.com/docs/javascript).
```

```
import { defineCustomElements as defineCalciteElements } from "@esri/calcite-components/dist/loader";
import { defineCustomElements as defineCodingElements } from "@arcgis/coding-components/dist/loader";

// define custom elements in the browser, and load the assets from the CDN
defineCalciteElements(window, { resourcesUrl: "https://js.arcgis.com/calcite-components/2.4.0/assets" });
defineCodingElements(window, { resourcesUrl: "https://js.arcgis.com/coding-components/next/assets" });
```

> We use [`src/index.js`](./src/index.js) to load our data, define our custom elements, and utilize various kinds of properties in the editor. It is a must to define both the Calcite and Coding elements.

Reviewer: ```suggestion
Use [`src/index.js`](./src/index.js) to load the data, define custom elements, and utilize various properties in the editor. Both the Calcite and Coding elements must be defined.
```
